cdap-docs/faq/source/cdap-startup.rst

```diff
@@ -0,0 +1,226 @@
+.. TBC.
+
+
+The CDAP UI is showing a message "namespace cannot be found".
+-------------------------------------------------------------
+This is indicative that the UI cannot connect to the CDAP system service containers running in YARN.
+
+- First, check if the CDAP Master service container shows as RUNNING in the YARN ResourceManager UI.
+
+- If this doesn't resolve the issue, then it means the CDAP system services were unable to launch.
+  Ensure :ref:`YARN has enough spare memory and vcore capacity <faq-installation-startup-memory-core-requirements>`.  
+  CDAP attempts to launch between 8 and 11 containers, depending on the configuration. Check
+  the master container (Application Master) logs to see if it was able to launch all containers.
+
+- If it was able to launch all containers, then you may need to further check the launched container logs for any errors.
+
+
+.. CDAP UI shows a session time out
+.. --------------------------------
+.. TBC.
+
+
+I don't see the CDAP Master service on YARN.
+--------------------------------------------
+- Ensure that the node where CDAP is running has a properly configured YARN client.
+- Ensure :ref:`YARN has enough memory and vcore capacity <faq-installation-startup-memory-core-requirements>`.
+
+
+My CDAP Master log shows permissions issues.
+--------------------------------------------
+Ensure that ``hdfs:///#{hdfs.namespace}`` and ``hdfs:///user/#{hdfs.user}`` exist and are owned by ``#{hdfs.user}``.
+
+In rare cases, until `CDAP-3817 <https://issues.cask.co/browse/CDAP-3817>`__ is resolved,
+ensure ``hdfs:///#{hdfs.namespace}/tx.snapshot`` exists and is owned by ``#{hdfs.user}``. 
+
+In any other case, the error should show which directory it is attempting to access. Don't
+hesitate to ask for help at the `cdap-user@googlegroups.com <https://groups.google.com/d/forum/cdap-user>`__.
+
+
+My CDAP Master log shows an error about the dataset service not being found.
+----------------------------------------------------------------------------
+If you see an error such as::
+
+    2015-05-15 12:15:53,028 - ERROR [heartbeats-scheduler:c.c.c.d.s.s.MDSStreamMetaStore$1@71] 
+    - Failed to access app.meta table co.cask.cdap.data2.dataset2.DatasetManagementException: 
+    Cannot retrieve dataset instance app.meta info, details: Response code: 407, 
+    message:'Proxy Authentication Required', 
+    body: '<HTML><HEAD> <TITLE>Access Denied</TITLE> </HEAD>
+    
+According to that log, this error can be caused by a proxy setting. CDAP services
+internally makes HTTP requests to each other; one example is the dataset service.
+Depending on your proxy and its settings, these requests can end up being sent to the
+proxy instead.
+
+One item to check is that your system's network setting is configured to exclude both
+``localhost`` and ``127.0.0.1`` from the proxy routing. If they aren't, the services will
+not be able to communicate with each other, and you'll see error messages such as these.
```

Comment:
How? At least on Linux, this isn't configurable easily on the user's side and must be done on the proxy side. Proxies are configured using only two environment variables, and exclusions aren't available.